src/pages/whitepaper.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+
+export default function Whitepaper(): JSX.Element {
+  return (
+    <Layout
+      title="Whitepaper"
+      description="Download the Yellow Network Whitepaper">
+      <main className="container margin-vert--xl">
+        <div className="row">
+          <div className="col col--8 col--offset-2 text--center">
+            <h1 className="hero__title margin-bottom--lg">Yellow Network Whitepaper</h1>
+            <p className="hero__subtitle margin-bottom--xl">
+              Understand the risks, utilities, and compliance framework of the Yellow token.
+            </p>
+            
+            <div className="card shadow--md padding--lg">
+              <div className="card__header">
+                <h3>Yellow MiCA Whitepaper</h3>
+              </div>
+              <div className="card__body">
+                <p>
+                  Our whitepaper provides a comprehensive overview of the Yellow Network protocol, 
+                  tokenomics, and regulatory compliance under MiCA.
+                </p>
+              </div>
+              <div className="card__footer">
+                <Link
+                  className="button button--primary button--lg"
+                  to="/assets/YELLOW_MiCA_White_Paper_v.1.1.pdf"
```

Comment:
this should be updated to point to the paper's latest revision

Reply:
Taken a pull from the master to get the updated file and updated the file name here.
[5c3340a071ed74079462e5fb4a526a4c41e9030f](https://github.com/layer-3/docs/pull/91/commits/5c3340a071ed74079462e5fb4a526a4c41e9030f)